design-tokens/tokens/layer1/base.json

```diff
@@ -1,109 +1,129 @@
+        "value": "0.25rem",
         "type": "dimension"
       },
-      "md": {
-        "value": "{size.dimension.sm} * {size.dimension.scale}",
+      "75": {
+        "value": "0.375rem",
         "type": "dimension"
       },
-      "lg": {
-        "value": "{size.dimension.md} * {size.dimension.scale}",
+      "100": {
+        "value": "0.5rem",
         "type": "dimension"
       },
-      "xl": {
-        "value": "{size.dimension.lg} * {size.dimension.scale}",
+      "125": {
+        "value": "0.75rem",
         "type": "dimension"
-      }
-    },
-    "spacing": {
-      "xs": {
-        "value": "{size.dimension.xs}",
-        "type": "spacing"
       },
-      "sm": {
-        "value": "{size.dimension.sm}",
-        "type": "spacing"
+      "150": {
+        "value": "1.25rem",
+        "type": "dimension"
       },
-      "md": {
-        "value": "{size.dimension.md}",
-        "type": "spacing"
+      "200": {
+        "value": "1rem",
+        "type": "dimension"
       },
-      "lg": {
-        "value": "{size.dimension.lg}",
-        "type": "spacing"
+      "300": {
+        "value": "1.5rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "{size.dimension.xl}",
-        "type": "spacing"
+      "400": {
+        "value": "2rem",
+        "type": "dimension"
       },
-      "multi-value": {
-        "value": "{size.dimension.sm} {size.dimension.xl}",
-        "type": "spacing",
-        "description": "You can have multiple values in a single spacing token. Read more on these: https://docs.tokens.studio/available-tokens/spacing-tokens#multi-value-spacing-tokens"
-      }
-    },
-    "borderRadius": {
-      "sm": {
-        "value": "4",
-        "type": "borderRadius"
+      "500": {
+        "value": "2.5rem",
+        "type": "dimension"
       },
-      "lg": {
-        "value": "8",
-        "type": "borderRadius"
+      "600": {
+        "value": "3rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "16",
-        "type": "borderRadius"
+      "800": {
+        "value": "4rem",
+        "type": "dimension"
+      }
+    },
+    "sizes": {
+      "global": {
+        "xs": {
+          "value": "0.25rem",
+          "type": "dimension"
+        }
       },
-      "multi-value": {
-        "value": "{size.borderRadius.sm} {size.borderRadius.lg}",
-        "type": "borderRadius",
-        "description": "You can have multiple values in a single radius token. Read more on these: https://docs.tokens.studio/available-tokens/border-radius-tokens#single--multiple-values"
+      "icons": {
+        "sm": {
+          "value": "1rem",
+          "type": "dimension"
+        },
+        "md": {
+          "value": "1.25rem",
+          "type": "dimension"
+        },
+        "lg": {
+          "value": "1.5rem",
+          "type": "dimension"
+        }
       }
     }
   },
-  "opacity": {
-    "low": {
-      "value": "10%",
-      "type": "opacity"
+  "borderRadius": {
```

Comment:
borderRaidus are in rem in the current tailwind config, should it be the same here?
Or should it be relative to the element it's applied to?

sm: 2 (px?) is probably small for something that has 100px size but it's huge for something that has a 4px size.

So either it should be relative or the name should not convey a meaning of small, medium, large and just be numbers like for our spacings.

Reply:
great point. I'll update this to relative values.

Reply:
Actually, I was reading more about it, and I think we should keep the static pixel values, and instead change the token name to an abstract value.